dojo/api_v2/serializers.py

```diff
@@ -388,6 +390,65 @@ class BurpRawRequestResponseSerializer(serializers.Serializer):
     req_resp = RequestResponseSerializerField(required=True)
 
 
+class BurpRawRequestResponseMultiSerializer(serializers.ModelSerializer):
+    burpRequestBase64 = serializers.CharField()
+    burpResponseBase64 = serializers.CharField()
+
+    def get_stringrequest(self, obj) -> str:
+        result = ""
+        if isinstance(obj, dict):
+            result = obj.get("string_request", "")
+        elif obj.string_request:
+            result = obj.string_request
+
+        return result
+
+    def get_stringresponse(self, obj) -> str:
+        result = ""
+        if isinstance(obj, dict):
+            result = obj.get("string_response", "")
+        elif obj.string_response:
+            result = obj.string_response
+
+        return result
```

Comment:
```suggestion
```
This does not look used anymore. Testing locally and it appears to work fine without it